content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -90,6 +90,10 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
 kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
 kubectl apply -f https://git.io/vPieo          # create resource(s) from url
 kubectl create deployment nginx --image=nginx  # start a single instance of nginx
+kubectl create job hello                       # create a job which prints "hello world"
+--image=busybox -- echo "Hello World"                                    
+kubectl create cronjob hello --image=busybox   # create a cron job which prints "hello world" for every minute
+--schedule="*/1 * * * *" -- echo "Hello World"       
```

Comment:
nits:  please consider indent line 93 and 96 by 2+ spaces so users can easily see those lines are not standalone commands;